frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.effects.ts

```diff
@@ -36,9 +37,16 @@ export class FeatureFlagsEffects {
         this.store$.pipe(select(selectTotalFlags)),
         this.store$.pipe(select(selectSearchKey)),
         this.store$.pipe(select(selectSortKey)),
-        this.store$.pipe(select(selectSortAs))
+        this.store$.pipe(select(selectSortAs)),
+        this.store$.pipe(select(selectIsAllFlagsFetched))
       ),
-      filter(([fromStarting, skip, total]) => skip < total || total === null || fromStarting),
+      filter(([fromStarting, skip, total, searchKey, sortKey, sortAs, isAllFlagsFetched]) => {
+        if (isAllFlagsFetched) {
```

Comment:
it appears like we don't set the `isLoadingFeatureFlags` to true until after this, it seems like this should suffice like the other filters that cancel this action though, do this work?

```javascript
filter(([fromStarting, skip, total, searchKey, sortKey, sortAs, isAllFlagsFetched]) => {
  return !isAllFlagsFetched || skip < total || total === null || fromStarting;
})
```

Reply:
No, actually we do set `isLoadingFeatureFlags` to **true** in the reducer on `actionFetchFeatureFlags`, which is why I explicitly made `isLoadingFeatureFlags` to **false** in the effect when all flags are fetched.

So either we can :
- set `isLoadingFeatureFlags` to **false** explicitly in Effects
or
-  remove state handling on `actionFetchFeatureFlags` in Reducer